src/main/java/de/storchp/opentracks/osmplugin/dashboardapi/ChairLift.java

```diff
@@ -0,0 +1,92 @@
+package de.storchp.opentracks.osmplugin.dashboardapi;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Bounds {
+    public double minlat;
+    public double minlon;
+    public double maxlat;
+    public double maxlon;
+
+    // Constructor
+    public Bounds(double minlat, double minlon, double maxlat, double maxlon) {
+        this.minlat = minlat;
+        this.minlon = minlon;
+        this.maxlat = maxlat;
+        this.maxlon = maxlon;
+    }
+
+    public Bounds() {
+
+    }
+}
+
+public class ChairLift {
+
+    private List<ChairLiftElements> chairLifts = new ArrayList<>();
+
+    private static ChairLift instance = null;
+
+    private ChairLift() {
+    }
+
+    // Static method to create instance of Singleton class
+    public static synchronized ChairLift getInstance() {
+        if (instance == null)
+            instance = new ChairLift();
+
+        return instance;
+    }
+
+    public List<ChairLiftElements> getChairLifts() {
+        return chairLifts;
+    }
+
+    public void addChairLiftData(ChairLiftElements chair) {
+        chairLifts.add(chairLift);
+    }
```

Comment:
If screen gets changed, system will call API again. In that case it will append chair lift. So call clearData() function in starting of API call so it clears old data.